components/reddit/reddit.app.mjs

```diff
@@ -18,7 +81,10 @@ module.exports = {
         for (const subreddit of results) {
           options.push({
             label: subreddit.title,
-            value: subreddit.displayName,
+            value: {
+              label: subreddit.title,
+              value: subreddit.displayName,
+            },
```

Comment:
Since the value for the `subreddit` prop (using structured mode) has changed, I think we'll need to update how `this.subreddit` is used in `sources/new-links-on-a-subreddit.mjs`, `sources/new-hot-posts-on-a-subreddit.mjs`, and `sources/new-comments-on-a-post.mjs`.